features/step_definitions/logging.rb

```diff
@@ -1389,3 +1389,25 @@ def check_log(logs, errors, exceptions)
     end
   end
 end
+
+Given /^I get storageclass from cluster and store it in the#{OPT_SYM} clipboard$/ do | sc |
+  sc = 'sc' unless sc
+  has_default_sc = false
+  _sc = ''
+  storage_classes = BushSlicer::StorageClass.list(user: user)
+  raise "Unable to get storage class " unless storage_classes.count > 0
+  storage_classes.each do | storage_class |
+    if storage_class.default?
+      has_default_sc = true
+      _sc = storage_class
+      break
+    end
+  end
+
+  if !has_default_sc
+    _sc = storage_classes.first
```

Comment:
@QiaolingTang, so if there's no storage class with type `default`, this code will assign the first storage_class that it finds and treat it as `default`.  I'm not certain that's the correct approach.  Should it return `nil` instead?  WDTY @duanwei33?

Reply:
Yes.

Reply:
> Should it return nil instead?

This is what we already have, but this makes my cases fail when there are some storage classes in the cluster and no storage class is marked as default.

Reply:
@QiaolingTang Hello, it might help somehow, but I need to remind that when our cases are executed in parallel, (and when there is no default storageclass), the first storageclass might be created by storage test case, and if it is for negative test, it may also cause failure. 
But I agree it is better than the previous one, we could use this and see if it has a good pass rate. If not, please consider to move it to ginkgo and skip it when there is no default storageclass. Or could we do the same (skip the case) in the cucushift, @pruan-rht ?

Reply:
@duanwei33 @QiaolingTang ok, let's merge it and see.  In cucushift, we could check that there is a default storageclass first before running anything and fail/skipp the test if there is no `default`